element_zstack/volume.py

```diff
@@ -114,13 +127,36 @@ def make(self, key):
                 px_width=volume_data.shape[2],
                 px_height=volume_data.shape[1],
                 px_depth=volume_data.shape[0],
-                depth_mean_brightness=volume_data.mean(axis=(1, 2))
+                depth_mean_brightness=volume_data.mean(axis=(1, 2)),
             )
         )
 
 
+class VoxelSize(dj.Manual):
+    definition = """
+    -> Volume
+    ---
+    width: float # voxel size in mm in the x dimension
+    height: float # voxel size in mm in the y dimension
+    depth: float # voxel size in mm in the z dimension
+    """
```

Comment:
Can these attributes be added to `Volume` table?  Is this information in the tiff header?

Reply:
Those info may or may not be available in the header, that is why it is designed as a separate table. Users may need to specify these info separately by manually insert into this table.

Reply:
Exactly what Thinh said above. If, in the future, we receive enough cases from users of this element stating that the information can be easily located within the tiff header, we will update the design to ingest that information automatically. For now, we're opting to keep this as a manual table.

Reply:
Thanks.  Sounds good.